tests/test.c.gcc-13.2.0-preproc

```diff
@@ -0,0 +1,743 @@
+# 0 "tests/test.c"
```

Comment:
would it be possible to have a clang preproc file too ?

Reply:
For this particular test, preprocessor output was identical. I'm not exactly sure how complex the test would need to grow to get a different preprocessor output. Does anyone have an example? I'll try to look into it if not.